indico/modules/receipts/default_templates/attendance/template.html

```diff
@@ -0,0 +1,103 @@
+{% set personal_data = registration.personal_data %}
+{% set today = now_utc() %}
+
+<h1>{{ custom_fields.document_title }}</h1>
+
+<aside id="title">
+  <div class="title">
+    <!-- Title -->
+    <h2>{{ event.title }}</h2>
+    <!-- Date(s) -->
+    {% if event.start_dt.date() != event.end_dt.date() %}
+      {{ format_interval(event.start_dt, event.end_dt, format='dMMMMy') }}
+    {% else %}
+      {{ event.start_dt | format_date('dd MMM YYYY') }}
+    {% endif %}
+
+    <!-- Venue -->
+    {% if custom_fields.venue %}
+      - {{ custom_fields.venue }}
+    {% elif event.venue_name %}
+      - {{ event.venue_name }}
+    {% endif %}
+
+    {% if custom_fields.add_url %}
+      <p>{{ event.url }}</p>
+    {% endif %}
+  </div>
+
+  <!-- Logo -->
+  {% if custom_fields.logo %}
+    <img src="{{ custom_fields.logo }}">
+  {% endif %}
+</aside>
+
+<aside id="addresses">
+  <!-- Address of organizer -->
+  <address id="from">
+    {{ custom_fields.address_from }}
+  </address>
+</aside>
+
+{% set start_date = event.start_dt | format_date('dd.MM.YYYY') %}
+{% set end_date = event.end_dt | format_date('dd.MM.YYYY') %}
+{%- if event.start_dt.date() != event.end_dt.date() %}
+  {% set date_range = 'between {} and {}'.format(start_date, end_date) %}
+{%- else %}
+  {% set date_range = 'on {}'.format(event.start_dt | format_date('dd.MM.YYYY')) %}
+{%- endif -%}
+    {%- if custom_fields.show_affiliation and personal_data.affiliation %}
```

Comment:
```suggestion
{%- if custom_fields.show_affiliation and personal_data.affiliation %}
```

This doesn't look like it's supposed to be indented.